xbmc/Application.cpp

```diff
@@ -2796,13 +2796,43 @@ bool CApplication::PlayFile(CFileItem item, const std::string& player, bool bRes
   if (!bRestart)
   {
     // bRestart will be true when called from PlayStack(), skipping this block
-    m_appPlayer.SetPlaySpeed(1);
+    if (item.IsVideo())
+    {
+      // we need to check if a video lock is set up
+      if (WINDOW_VIDEO_NAV != CServiceBroker::GetGUI()->GetWindowManager().GetActiveWindow())
+      {
+        if (CServiceBroker::GetSettingsComponent()
+                ->GetProfileManager()
+                ->GetCurrentProfile()
+                .videoLocked())
+        {
+          if (!g_passwordManager.IsMasterLockUnlocked(false))
+          {
+            const CVideoInfoTag *tag = item.GetVideoInfoTag();
```

Comment:
> Left-align `*` and `&` to the base type they modify.

https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md#52-pointer-and-reference-types